streams.json

```diff
@@ -3320,55 +4395,65 @@
               "tsaloranta@gmail.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/FasterXML/java-classmate",
+          "repository_url": "https://github.com/FasterXML/java-classmate.git",
           "codebase": "master",
           "tag": "classmate-1.3.3",
+          "group_id": "com.fasterxml",
+          "artifact_id": "classmate",
           "version": "1.3.3.redhat-1",
-          "gav": "com.fasterxml",
+          "gav": "classmate:com.fasterxml:1.3.3.redhat-1",
           "comment": ""
         },
         {
           "component_name": "java-getopt",
           "contacts": [],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/arenn/java-getopt",
+          "repository_url": "https://github.com/arenn/java-getopt.git",
           "codebase": "",
-          "tag": "1.0.13.redhat-5",
+          "tag": "",
+          "group_id": "gnu.getopt",
+          "artifact_id": "java-getopt",
           "version": "1.0.13.redhat-5",
-          "gav": "gnu.getopt",
-          "comment": " Master is the branch, no tags, just binary"
+          "gav": "java-getopt:gnu.getopt:1.0.13.redhat-5",
+          "comment": "TAG not found.  Master is the branch, no tags, just binary"
         },
         {
           "component_name": "jaxb~version2",
```

Comment:
This one is weird. I would suggest linking it with org.glassfish.jaxb:jaxb-core or leaving it out of the final list.